src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread ([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
Sorry, can't understand your motivation. You want to add a hint in UI, suggest a solution which does not solve the problem (why would a script execution time be different from that of a command?), create a work for translators. It can't be just to escape from typing two lines of code, can it?